flexinput/src/main/java/com/lytefast/flexinput/adapters/PhotoCursorAdapter.kt

```diff
@@ -55,12 +60,12 @@ class PhotoCursorAdapter(private val contentResolver: ContentResolver,
 
   override fun onBindViewHolder(holder: ViewHolder, position: Int, payloads: MutableList<Any>) {
     payloads
-      .firstOrNull { it is SelectionCoordinator.SelectionEvent<*> }
-      ?.let { it as? SelectionCoordinator.SelectionEvent<*> }
-      ?.also {
-        holder.setSelected(it.isSelected, isAnimationRequested = true)
-        return
-      }
+        .firstOrNull { it is SelectionCoordinator.SelectionEvent<*> }
```

Comment:
revert if there are no changes

Reply:
This is an uphill battle I won't fight. I updated autoformatter to match the proper settings of this project.

Reply:
It's distracting from the important parts here. Let me format patch this in a different commit first then...
Try rebasing now.